app/client/ui/WelcomeCoachingCall.ts

```diff
@@ -77,35 +78,39 @@ export function showWelcomeCoachingCall(triggerElement: Element, appModel: AppMo
           testId('popup-close-button'),
         ),
       ),
-      cssPopupTitle('Free Coaching Call', testId('popup-title')),
+      cssPopupTitle(t('free coaching call'),
+        dom.style('text-transform', 'capitalize'),
+        testId('popup-title')
+      ),
       cssPopupBody(
         cssBody(
           dom('div',
-            'Schedule your ', cssBoldText('free coaching call'), ' with a member of our team.'
+            t('Schedule your {{freeCoachingCall}} with a member of our team.',
+              {freeCoachingCall: cssBoldText('free coaching call')}
+            )
           ),
           dom('div',
-            "On the call, we'll take the time to understand your needs and "
-            + 'tailor the call to you. We can show you the Grist basics, or start '
-            + 'working with your data right away to build the dashboards you need.'
+            t("On the call, we'll take the time to understand your needs and tailor the call to you. \
+We can show you the Grist basics, or start working with your data right away to build the dashboards you need.")
           ),
         ),
         testId('popup-body'),
       ),
       cssPopupButtons(
         bigPrimaryButtonLink(
-          'Schedule Call',
+          t('Schedule Call'),
           dom.on('click', () => {
             dismissPopup(false);
             logTelemetryEvent('clickedScheduleCoachingCall');
           }),
           {
-            href: FREE_COACHING_CALL_URL,
+            href: getFreeCoachingCallUrl(),
```

Comment:
```suggestion
            href: getGristConfig().freeCoachingCallUrl,
```

This way, you don't need `getFreeCoachingCallUrl` in `gristUrls.ts`.